tests/Doctrine/Tests/ORM/Mapping/BasicInheritanceMappingTest.php

```diff
@@ -243,18 +243,26 @@ class EntitySubClass extends TransientBaseClass
 /** @MappedSuperclass */
 class MappedSuperclassBase
 {
-    /** @Column(type="integer") */
+    /**
+     * @var int
+     * @Column(type="integer")
+     */
     private $mapped1;
+
     /**
      * @var string
      * @Column(type="string")
      */
     private $mapped2;
+
     /**
+     * @var MappedSuperclassRelated1
      * @OneToOne(targetEntity="MappedSuperclassRelated1")
      * @JoinColumn(name="related1_id", referencedColumnName="id")
      */
     private $mappedRelated1;
+
+    /** @var mixed */
     private $transient;
 }
 class MappedSuperclassRelated1
```

Comment:
I'm surprised that the coding standard is not complaining about the missing new line between classes.

Reply:
It's surprising indeed, I'll add one in my next PR :)